testing-common/src/main/java/io/opentelemetry/instrumentation/testing/LibraryTestRunner.java

```diff
@@ -70,6 +70,11 @@ public void clearAllExportedData() {
     forceFlushCalled = false;
   }
 
+  @Override
+  public OpenTelemetrySdk getOpenTelemetry() {
```

Comment:
We should almost never need the SDK in tests (instrumentation is supposed to avoid the SDK) so I think the old name was better when the return type is `OpenTelemetrySdk`. So I figured for the rare cases (aws lambda, spring) where we need the SDK, calling into `testRunner.openTelemetrySdk` in library tests was OK, while in the more common case we had `LibraryTestTrait.getOpenTelemetry()`. Do we need to make a change here or is it ok to revert it?

Reply:
I've added back the `getOpenTelemetrySdk()` method to the library runner and reverted back the AWS tests.